parser/openfoodfacts_taxonomy_parser/parser.py

```diff
@@ -0,0 +1,283 @@
+from neo4j import GraphDatabase
+import re,unicodedata,unidecode
+from .exception import DuplicateIDError
+
+
+class Parser:
+
+    def __init__(self,uri="bolt://localhost:7687"):
+        self.driver = GraphDatabase.driver(uri)
+        self.session = self.driver.session() #Doesn't create error even if there is no active database
+
+    def create_headernode(self,header):
+        """ create the node for the header """
+        query = """
+                CREATE (n:TEXT {id: '__header__' })
+                SET n.preceding_lines= $header
+                SET n.src_position= 1
+            """
+        self.session.run(query,header=header)
+
+    def create_node(self,data):
+        """ run the query to create the node with data dictionary """
+        position_query = """
+            SET n.previous_block = $previous_block
```

Comment:
@BryanH01 @alexgarel Is this property called "previous_block" or "is_before"?

Reply:
Oops sorry my bad, i forgot to change it

Reply:
as you want @BryanH01